app/routes.py

```diff
@@ -1,2 +1,33 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify
 
+class Planet:
+    def __init__(self, id, name, description, matter):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.matter = matter
+
+
+planets = [Planet(1, "Mercury", "small and red", "solid"),     
+Planet(5, "Jupiter", "big and swirly", "gaseous"), 
+Planet(6, "Saturn", "rings and swirls", "gaseous")]
+
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@planets_bp.route("", methods=["GET"])
+def get_all_planets():
+    planet_list = []
+    for planet in planets:
+        planet_list.append(vars(planet))
```

Comment:
Great use of vars!

Reply:
I just learned that `vars` will cause problems when we have a class that connects to our database. We can achieve the same effect by writing a `to_json(self)` instance method on the `Planet` class.